lib/list.gd

```diff
@@ -2376,36 +2376,22 @@ DeclareGlobalFunction( "ListWithIdenticalEntries" );
 ##
 #F  PlainListCopy( <list> ) . . . . . . . .  make a plain list copy of a list
 ##
+##  <#GAPDoc Label="PlainListCopy">
 ##  <ManSection>
 ##  <Func Name="PlainListCopy" Arg='list'/>
 ##
 ##  <Description>
+##  This operation returns a list equal to its argument, in a plain list
```

Comment:
`PlainListCopy` is currently a function (although maybe it _should_ be an operation? It's not important though)
```suggestion
##  This function returns a list equal to its argument, in a plain list
```